docker-app/qfieldcloud/core/views/files_views.py

```diff
@@ -158,6 +158,15 @@ def post(self, request, projectid, filename, format=None):
             )
 
         request_file = request.FILES.get("file")
+
+        file_size_mb = file_size_bytes(request_file) / 1024 / 1024
```

Comment:
```suggestion
        file_size_mb = request_file.size / 1024 / 1024
```
[Don't you have something like this?](https://docs.djangoproject.com/en/4.0/ref/files/uploads/#django.core.files.uploadedfile.UploadedFile.size)

Reply:
Indeed (got confused because that doesn't exist for io.BytesIO, but here we deal with a django UploadedFile)